src/collection.js

```diff
@@ -158,24 +160,26 @@ export default class Collection {
    * Adds a record to the local database.
    *
    * Options:
-   * - {Boolean} synced     Sets record status to "synced" (default: false).
-   * - {Boolean} forceUUID  Enforces record creation using any provided UUID
-   *                        (default: false).
+   * - {Boolean} synced   Sets record status to "synced" (default: false).
+   * - {Boolean} forceId  Forces the id field from the record to be used, instead
```

Comment:
`forceID` doesn't seem self-explanatory to me (without reading the documentation about it, I have no idea what this option does). Should we rename it to something different? `useRecordID`?